data/anaconda.conf

```diff
@@ -302,6 +302,10 @@ password_policies =
     user (quality 1, length 6, empty)
     luks (quality 1, length 6)
 
+# Show 64k memory page kernel on ARM.
+show_arm_64k_kernel = False
```

Comment:
Please, add an extra line between the two new options.

Reply:
If we are going to detect the packages than we don't need them?